spec/features/notifications/reminder_mail_spec.rb

```diff
@@ -9,10 +9,10 @@
   let(:work_package) { create(:work_package, project:) }
   let(:watched_work_package) { create(:work_package, project:, watcher_users: [current_user]) }
   let(:involved_work_package) { create(:work_package, project:, assigned_to: current_user) }
-  # The run_at time of the delayed job used for scheduling the reminder mails
+  # GoodJob::Job#scheduled_at is used for scheduling the reminder mails
```

Comment:
This goes back to the comment further up. Perhaps we could just call the method `job_scheduled_at` in `ApplicationJob` and reference that here?